### ISSUE TITLE
Test integration logging not working as expected 

### ISSUE THREAD

**Issue by GregHanson** [istio/istio#24376]
**Bug description**

There are two logging scopes defined here, `CI` and `Framework`: https://github.com/istio/istio/blob/master/pkg/test/scopes/scopes.go

Should these two items be combined?  Or what is the difference between them?  While running integration tests I found I could only configure one of these loggers:
```
go test  ...   --log_output_level=debug,tf:debug,ci:debug
```
`tf` (or Framework) goes to debug as expected, but `ci` output is still nowhere to be seen

**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ X ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Affected features (please put an X in all that apply)**

[ ] Multi Cluster
[ ] Virtual Machine
[ ] Multi Control Plane

**Expected behavior**

Should be able to configure log level for running integration tests locally

**Steps to reproduce the bug**

```
go test  ./tests/integration/multimaster/...   --log_output_level=debug,tf:debug,ci:debug
```

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
master branch

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**
IKS

**Comment by GregHanson** [comment]
cc @howardjohn

**Comment by howardjohn** [comment]
Its based entirely on the istio.test.ci flag: https://github.com/istio/istio/blob/65bd2ac632cbd6ec8e3fd85bb9874f685aea0cfd/pkg/test/framework/logging.go#L38

**Comment by GregHanson** [comment]
Ok then the question is do we need both `Framework` and `CI` loggers? And when should one be used over the other?  I don't think the usage is clear.

Also, as it stands now, `CI` is either None or Info level logging:
```
	if ciMode {
		o.SetOutputLevel(scopes.CI.Name(), log.InfoLevel)
	} else {
		o.SetOutputLevel(scopes.CI.Name(), log.NoneLevel)
	}
```
Which means there is nothing stopping someone from adding a Debug level log message which will never be displayed:
```
$ grep -rIn "CI.Debug" .
./tests/integration/telemetry/dashboard_test.go:206:			scopes.CI.Debugf("Filtered out metric '%v', but got samples: %v", query, value)
```

**Comment by howardjohn** [comment]
Yeah I would agree we can probably just remove the CI one in favor of framework. I don't know any users that don't want the "ci" mode

**Comment by GregHanson** [comment]
This might be a Developer Experience item, cc @sdake

From the [wiki](https://github.com/istio/istio/tree/master/tests/integration#enabling-ci-mode)
>The flag is not enabled by default to provide a better U/X when running tests locally (i.e. additional logging can clutter test output and error dumping can take quite a while)

Personally, if I am writing a test, I would prefer to have the logs if a test fails, rather than re-running a lengthy integration test a second time with an additional flag.
